src/Form/ManagementSMTPType.php

```diff
@@ -46,7 +47,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'required' => false, ]);
 
         $builder->add('user', TextType::class, ['required' => false]);
-        $builder->add('password', TextType ::class, ['required' => false]);
+        $builder->add('password', PasswordType ::class, ['required' => false]);
```

Comment:
@AlexMyddleware Did we get confirmation about the ability to leave the password field empty when using this FormType?
I think some SMTPs (particularly on localhost) allow you to have an empty password to log in, so if I'm not mistaken, we need to keep this option available for users who use an SMTP with an empty password, TBC.